src/reducers/shop.js

```diff
@@ -136,3 +137,43 @@ const quantityById = (state = INITIAL_CART.quantityById, action) => {
 }
 
 export default shop;
+
+const cartSelector = state => state.shop.cart;
+const productsSelector = state => state.shop.products;
+
+export const cartItemsSelector = createSelector(
+  cartSelector,
+  productsSelector,
+  (cart, products) => {
```

Comment:
In the interest of this being a learning template, what do you think about excessively documenting these? I.e. a generic comment about what a selector is, why these 5 are here, what each does maybe?

Reply:
Absolutely! I'll add some.

Reply:
Done!